src/cleo/commands/completions_command.py

```diff
@@ -258,11 +282,12 @@ def sanitize(s: str) -> str:
                 f"complete -c {script_name} -f -n '__fish{function}_no_subcommand' "
                 f"-a {command_name} -d '{sanitize(cmd.description)}'"
             )
+
             cmds_opts += [
-                f"# {command_name}",
+                f"# {cmd.name}",
                 *[
                     f"complete -c {script_name} -A "
-                    f"-n '__fish_seen_subcommand_from {sanitize(command_name)}' "
```

Comment:
just to make sure I understand, the syntax error was already fixed here with https://github.com/python-poetry/cleo/commit/6cc808dd97c811146d72d1c674cfbbdcaf103ce7 right? we just haven't had a release since then

Reply:
Well, the fix was making `'__fish_seen_subcommand_from \'command name\''` entry, while this fix changes it to being `'__fish_seen_subcommand_from "command name"'` which feels like a more "proper" way to nest quotes.